README.md

```diff
@@ -0,0 +1,34 @@
+# Playbooks
+This repository includes all the playbooks needed to prepare a server to run [TimeOverflow](https://github.com/coopdevs/timeoverflow).
+```
+playbooks/
+├── provision.yml
+└── sys_admins.yml
+```
+
+## sys_admins.yml
+This playbook will prepare the host to allow access to all the system administrators.
+
+The first time you run it against a brand new host you need to run it as `root` user.
+You'll also need passwordless SSH access to the `root` user.
+```
+ansible-playbook playbooks/sys_admins.yml --limit=dev -u root
```

Comment:
What about parametrizing the --limit to make the user understand that depends on the envieonment? Something like <environment_name>?